core/test/XiosReadWrite_test.cpp

```diff
@@ -128,6 +130,11 @@ Xios setupXiosHandler(int dim, bool read)
     xios_handler.fileAddField(fileId, fieldId);
 
     xios_handler.close_context_definition();
+    // FIXME: Why do we need to re-set the calendar timestep and start here?
+    //        These are already set in the Xios constructor and the test fails if the following two
+    //        lines are moved above the close_context_definition.
+    xios_handler.setCalendarTimestep(Duration("P0-0T01:30:00"));
+    xios_handler.setCalendarStart(TimePoint("2023-03-17T17:11:00Z"));
```

Comment:
I considered updating to XIOS3 but found that some of the demos hang with that so figured it'd be better to address this specific issue first.

Fixed the issue above locally with a hack as in https://github.com/nextsimhub/nextsimdg/pull/757/commits/fe1c9b22c4e30054fd44040c2fccdd6ef8d181a2. My latest commit triggered the Docker image to be rebuilt so hopefully it will work in there, too.

Reply:
Okay so initially the test suite with XIOS failed but then once the Docker image had built, I re-triggered it and it passed!